### PR TITLE
Packages

### PR DESCRIPTION
**Pull request by eggrobin**
Address #602 for the files in `physics`.

### CODE REVIEW

File: physics/body_centered_non_rotating_dynamic_frame_test.cpp [hunk 1]
```diff
@@ -41,8 +48,6 @@ using ::testing::IsNull;
 using ::testing::Lt;
 using ::testing::Not;
 
-namespace physics {
-
 namespace {
```
Comment (pleroy) [inline physics/body_centered_non_rotating_dynamic_frame_test.cpp]: Remove this namespace?

File: physics/discrete_trajectory.hpp [hunk 2]
```diff
@@ -25,12 +27,14 @@ using quantities::Acceleration;
...
 
-namespace physics {
-
 template<typename Frame>
 class DiscreteTrajectory;
 
-namespace internal {
+}  // namespace internal_discrete_trajectory
+
+namespace internal_forkable {
```
Comment (pleroy) [inline physics/discrete_trajectory.hpp]: Comment why to have to reopen this.

File: physics/discrete_trajectory_test.cpp [hunk 3]
```diff
@@ -572,22 +572,22 @@ TEST_F(DiscreteTrajectoryTest, ForgetAfter) {
...
   fork->ForgetAfter(t2_);
   positions = Positions(*fork);
   velocities = Velocities(*fork);
   times = Times(*fork);
-  EXPECT_THAT(positions, ElementsAre(testing::Pair(t1_, q1_),
-                                     testing::Pair(t2_, q2_)));
-  EXPECT_THAT(velocities, ElementsAre(testing::Pair(t1_, p1_),
-                                      testing::Pair(t2_, p2_)));
+  EXPECT_THAT(positions, ElementsAre(Pair(t1_, q1_),
+                                     Pair(t2_, q2_)));
```
Comment (pleroy) [inline physics/discrete_trajectory_test.cpp]: This and the next would fit on one line I think.

File: physics/discrete_trajectory_test.cpp [hunk 4]
```diff
@@ -640,30 +640,30 @@ TEST_F(DiscreteTrajectoryTest, ForgetBeforeSuccess) {
...
   massive_trajectory_->ForgetBefore(t2_);
   positions = Positions(*massive_trajectory_);
   velocities = Velocities(*massive_trajectory_);
   times = Times(*massive_trajectory_);
-  EXPECT_THAT(positions, ElementsAre(testing::Pair(t2_, q2_),
-                                     testing::Pair(t3_, q3_)));
-  EXPECT_THAT(velocities, ElementsAre(testing::Pair(t2_, p2_),
-                                      testing::Pair(t3_, p3_)));
+  EXPECT_THAT(positions, ElementsAre(Pair(t2_, q2_),
+                                     Pair(t3_, q3_)));
```
Comment (pleroy) [inline physics/discrete_trajectory_test.cpp]: Same.

File: physics/dynamic_frame_test.cpp [hunk 5]
```diff
@@ -9,16 +9,23 @@
...
 using quantities::GravitationalParameter;
 using quantities::Sqrt;
+using quantities::si::Metre;
+using quantities::si::Second;
 using testing_utilities::AlmostEquals;
 using testing_utilities::Componentwise;
 
-namespace physics {
-
 namespace {
```
Comment (pleroy) [inline physics/dynamic_frame_test.cpp]: Remove?

File: physics/ephemeris_test.cpp [hunk 6]
```diff
@@ -63,8 +69,6 @@ using ::testing::Gt;
 using ::testing::Lt;
 using ::testing::Ref;
 
-namespace physics {
-
 namespace {
```
Comment (pleroy) [inline physics/ephemeris_test.cpp]: Remove?

File: physics/frame_field.hpp [hunk 7]
```diff
@@ -25,6 +25,11 @@ using FrameField =
 template<typename Frame>
 FrameField<Frame> CoordinateFrame();
 
+}  // namespace internal_frame_field;
+
+using internal_frame_field::FrameField;
+using internal_frame_field::CoordinateFrame;
```
Comment (pleroy) [inline physics/frame_field.hpp]: Alphabetize.

File: physics/dynamic_frame.hpp [hunk 8]
```diff
@@ -58,6 +60,11 @@ class DynamicFrame {
                       serialization::DynamicFrame const& message);
 };
 
+}  // namespace internal_dynamic_frame
+
+using internal_dynamic_frame::Frenet;
+using internal_dynamic_frame::DynamicFrame;
```
Comment (pleroy) [inline physics/dynamic_frame.hpp]: Alphabetize.

File: physics/rigid_motion.hpp [hunk 9]
```diff
@@ -75,6 +76,11 @@ RigidMotion<FromFrame, ToFrame> operator*(
     RigidMotion<ThroughFrame, ToFrame> const& left,
     RigidMotion<FromFrame, ThroughFrame> const& right);
 
+}  // namespace internal_rigid_motion
+
+using internal_rigid_motion::RigidTransformation;
+using internal_rigid_motion::RigidMotion;
```
Comment (pleroy) [inline physics/rigid_motion.hpp]: Alphabetize.

File: physics/barycentric_rotating_dynamic_frame_test.cpp [hunk 10]
```diff
@@ -50,8 +52,6 @@ using ::testing::Return;
 using ::testing::StrictMock;
 using ::testing::_;
 
-namespace physics {
-
 namespace {
```
Comment (pleroy) [inline physics/barycentric_rotating_dynamic_frame_test.cpp]: Remove?

File: physics/rigid_motion.hpp (hunk 9 above)
Comment (eggrobin) [inline physics/rigid_motion.hpp]: Done.

File: physics/dynamic_frame.hpp (hunk 8 above)
Comment (eggrobin) [inline physics/dynamic_frame.hpp]: Done.

File: physics/frame_field.hpp (hunk 7 above)
Comment (eggrobin) [inline physics/frame_field.hpp]: Done.

File: physics/discrete_trajectory_test.cpp (hunk 4 above)
Comment (eggrobin) [inline physics/discrete_trajectory_test.cpp]: Done.

File: physics/discrete_trajectory_test.cpp (hunk 3 above)
Comment (eggrobin) [inline physics/discrete_trajectory_test.cpp]: Done.

File: physics/discrete_trajectory.hpp (hunk 2 above)
Comment (eggrobin) [inline physics/discrete_trajectory.hpp]: Done.

**Pull request merged**